src/main/groovy/life/qbic/datamodel/datasets/datastructure/folders/nanopore/MeasurementFolder.groovy

```diff
@@ -26,6 +26,12 @@ class MeasurementFolder extends DataFolder {
         validateName()
     }
 
+    /**
+     * Creates a new instance of a MeasurementFolder object
+     * @param relativePath The relative path of the folder
+     * @param children A list with child elements of the folder
+     * @return A new instance of a MeasurementFolder object
+     */
     static MeasurementFolder create(String name, String relativePath, List children) {
```

Comment:
In the other `create` methods you used to provide a `List<?>` here you only ask for a `List`. Is this intended?

Reply:
It is more a information for the compiler, not really a big thing. The `?` just states explicitely that we do not know the type during compile time.